### PR TITLE
Required changes for djaoapp #77

### PR DESCRIPTION
**Pull request by knivets**

### CODE REVIEW

File: saas/static/js/djaodjin-saas-vue.js [hunk 1]
```diff
@@ -1655,48 +1648,6 @@ Vue.component('user-typeahead', {
     }
 });
 
-// TODO it probably makes sense to use this component in other
-// places where user relations are needed
-Vue.component('user-relation', {
```
Comment (smirolo) [inline saas/static/js/djaodjin-saas-vue.js]: Why did this component get removed?

File: saas/static/js/djaodjin-saas-vue.js [hunk 2]
```diff
@@ -2156,7 +2107,6 @@ Vue.component('plan-subscriber-list', {
                             email: str,
                             full_name: str
                         }
-                        vm.modalShow();
```
Comment (smirolo) [inline saas/static/js/djaodjin-saas-vue.js]: Why did this statement get removed from the `plan-subscriber-list` component (which is used in both `templates/saas/profile/plans/subscribers.html` and `templates/saas/profile/subscribers.html`) and wasn't replaced by something like `vm.formVisible = true;` as done previously?

File: saas/static/js/djaodjin-saas-vue.js (hunk 1 above)
Comment (knivets) [reply saas/static/js/djaodjin-saas-vue.js]: I grepped and this component is not used anywhere.

File: saas/static/js/djaodjin-saas-vue.js (hunk 2 above)
Comment (knivets) [reply saas/static/js/djaodjin-saas-vue.js]: there is no such method in this component or any other mixin, so it was likely throwing exceptions anyway

File: saas/static/js/djaodjin-saas-vue.js (hunk 1 above)
Comment (knivets) [reply saas/static/js/djaodjin-saas-vue.js]: https://github.com/search?q=org%3Adjaodjin+user-relation&type=Code

**Comment by smirolo** [comment]
superceded by commit d212fee

**Pull request closed without merging**